phalcon/Mvc/Model.zep

```diff
@@ -3900,6 +3900,16 @@ abstract class Model extends AbstractInjectionAware implements EntityInterface,
                                         );
                                     }
 
+                                    /**
+                                     * Get actual values before comparison
+                                     */
+                                    if is_object(snapshotValue) && snapshotValue instanceof RawValue {
+                                        let snapshotValue = snapshotValue->getValue();
+                                    }
+                                    if is_object(value) && value instanceof RawValue {
```

Comment:
```suggestion
                                    if value instanceof RawValue {
```

Reply:
`instanceof` expects an object instance, but `value` can be other type